src/test/scala/jp/ne/opt/chronoscala/RichZonedDateTimeSpec.scala

```diff
@@ -10,11 +10,12 @@ object RichZonedDateTimeSpec extends Properties("RichZonedDateTime") with Gens {
     a <- zonedDateTimeGen
     b <- zonedDateTimeGen
     c <- zonedDateTimeGen
-  } yield (a, b, c)) { case (a, b, c) =>
-    val antisymmetry = !(a <= b && b <= a) || a == b
-    val transitivity = !(a <= b && b <= c) || a <= c
-    val totality = a <= b || b <= a
+  } yield (a, b, c)) {
+    case (a, b, c) =>
```

Comment:
I prefer `yield` and `case (a, b, c) =>` in one line.
(I don't know there is such preference or not)